database/migrations/default/1731003209538_rename_is_coa_roadway/down.sql

```diff
@@ -0,0 +1,148 @@
+                    and st_contains(geometry, new.position)
+                limit 1);
+        end if;
+
+        raise debug 'found location: % compared to previous location: %', new.location_id, old.location_id;
+        --
+        -- check if in austin full purpose jurisdiction
+        --
+        new.in_austin_full_purpose =  st_contains((select geometry from geo.atd_jurisdictions where id = 5), new.position);
+        raise debug 'in austin full purpose: % compared to previous: %', new.in_austin_full_purpose, old.in_austin_full_purpose;
+        --
+        -- get council district
+        --
+        new.council_district = (
+            select
+                council_district
+            from
+                geo.council_districts
+            where
+                st_contains(geometry, new.position)
+            limit 1);
+        raise debug 'council_district: % compared to previous: %', new.council_district, old.council_district;
+        --
+        -- get engineering area
+        --
+        new.engineering_area_id = (
+            select
+                engineering_area_id
+            from
+                geo.engineering_areas
+            where
+                st_contains(geometry, new.position)
+            limit 1);
+        raise debug 'engineering_area_id: % compared to previous: %', new.engineering_area_id, old.engineering_area_id;
+        --
+        -- get signal engineer area
+        --
+        new.signal_engineer_area_id = (
+            select
+                signal_engineer_area_id
+            from
+                geo.signal_engineer_areas
+            where
+                st_contains(geometry, new.position)
+            limit 1);
+        raise debug 'signal_engineer_area_id: % compared to previous: %', new.signal_engineer_area_id, old.signal_engineer_area_id;
+        --
+        -- get zip code
+        --
+        new.zipcode = (
+            select
+                zipcode
+            from
+                geo.zip_codes
+            where
+                st_contains(geometry, new.position)
+            limit 1);
+        raise debug 'zipcode: % compared to previous: %', new.zipcode, old.zipcode;
+        --
+        -- get apd sector
+        --
+        new.apd_sector_id = (
+            select
+                primary_key
+            from
+                geo.apd_sectors
+            where
+                st_contains(geometry, new.position)
+            limit 1);
+        raise debug 'apd_sector_id: % compared to previous: %', new.apd_sector_id, old.apd_sector_id;
+        --
+        -- check if is_non_coa_roadway
+        --
+        if (new.in_austin_full_purpose or new.rpt_city_id = 22) then
+            new.is_non_coa_roadway = st_contains((select geometry from geo.non_coa_roadways), new.position);
+            raise debug 'is_non_coa_roadway: % compared to previous: %', new.is_non_coa_roadway, old.is_non_coa_roadway;
+        end if;
```

Comment:
so previously if the city was updated to be outside of austin then no updates were happening to `is_non_coa_roadway` bc there is no `else` here? seems like a bug

Reply:
yep—it was!